engine/runtime-integration-tests/src/test/java/org/enso/logging/service/logback/telemetry/test/TestLogFormatter.java

```diff
@@ -0,0 +1,124 @@
+package org.enso.logging.service.logback.telemetry.test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.Matchers.nullValue;
+
+import org.enso.logging.service.logback.telemetry.LogFormatter;
+import org.enso.logging.service.logback.telemetry.LogMessage;
+import org.junit.Test;
+
+public class TestLogFormatter {
```

Comment:
These tests are (all) wrong. [The specification](https://github.com/enso-org/enso/issues/12464#issuecomment-2713545076) explicitly says:

> parameter names follow in a form of `param_name={}` and are separated by commas

E.g. we have to modify all the tests to use `{}`!